week1/1_Strings_and_Output.ipynb

```diff
@@ -213,7 +213,7 @@
    },
    "outputs": [],
    "source": [
-    "print('\"Why were you lurking under our window?\" \"Yes - yes, good point, Petunia! What were you doing under our windows, boy?\" \"Listening to the news,\" said Harry in a resigned voice. His aunt and uncle exchanged looks of outrage. \"Listening to the news! Again?\" \"Well, it changes every day, you see,\" said Harry.')"
+    "print('\"Listening to the news,\" said Harry in a resigned voice. His aunt and uncle exchanged looks of outrage. \"Listening to the news! Again?\" \"Well, it changes every day, you see,\" said Harry.')"
```

Comment:
I think the long example (and the buildup) adds some value.
Would you please rollback this specific change? All the others are terrific.